### PR TITLE
Update repo ReadMe.md file

### LINKED ISSUE #512: Update repo ReadMe.md file
In line with the issue available [here](https://github.com/BHoM/admin/issues/4), please update your repositories ReadMe.md file to match the template provided in the linked admin issue.

Please remember to change any instances of `Robot_Toolkit` in the template to your repository name.

I will comment separately with a specific AppVeyor badge link that should be used for this repo.

When you resolve this issue, please add myself and @al-fisher as reviewers.

### PR DESCRIPTION
**Pull request by adecler**
### Issues addressed by this PR

Closes #512

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,32 @@
-[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
-# BHoM
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/rw19y8e78jxd0r0d/branch/master?svg=true)](https://ci.appveyor.com/api/projects/status/grasshopper_toolkit/branch/master) [![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/Grasshopper_Toolkit/Grasshopper_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=119&branchName=master)
```
Comment (FraserGreenroyd) [inline README.md:1]: `119` definition ID is for compliance, `117` is for core, Azure badge should be `[Build Status](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=117&branchName=master)`

File: README.md [hunk 2]
```diff
@@ -1,54 +1,32 @@
...
-A great place to start is reading our Wiki [here](https://github.com/BHoM/documentation/wiki) including pages like the [Structure of the BHoM](https://github.com/BHoM/documentation/wiki/Structure-of-the-BHoM) and [Using the BHoM](https://github.com/BHoM/documentation/wiki/Using-the-BHoM).
+# Grasshopper_Toolkit
 
-## Quick start ##
+The Grasshopper_Toolkit provides a plugin that exposes the BHoM functionality inside Grasshopper.
 
-Try the [installer](http://bhom.xyz/assets/installers/v2.1/BHoM%20Alpha%20v2.1.0.5%20Installer.exe) and a selection of [sample scripts](https://github.com/BHoM/samples).
 
+### Known Versions of Software Supported
+Grasshopper for Rhino 5
```
Comment (al-fisher) [inline README.md:9]:
```suggestion
Grasshopper for Rhino 5  
```
Forced new line required for markdown

**Pull request merged**